config/application.yml.example

```diff
@@ -28,6 +28,8 @@ MAIL_DOMAIN: 'example.com'
 MAIL_PORT: 25
 SMTP_USERNAME: 'ofn'
 SMTP_PASSWORD: 'f00d'
+# MAILS_FROM: hello@example.com
+# MAIL_BCC: manager@example.com
```

Comment:
I guess you commented this out because they are not mandatory :thinking: . If that is the case, I'd explain why is that in a comment block preceding these lines. I'm afraid it might not be clear for new devs otherwise.